proto/axelarnet/v1beta1/tx.proto

```diff
@@ -28,7 +28,10 @@ message LinkResponse { string deposit_addr = 1; };
 message ConfirmDepositRequest {
   bytes sender = 1 [ (gogoproto.casttype) =
                          "github.com/cosmos/cosmos-sdk/types.AccAddress" ];
-  bytes tx_id = 2 [ (gogoproto.customname) = "TxID" ];
+  bytes tx_id = 2 [
+    (gogoproto.customname) = "TxID",
+    deprecated = true
+  ];
```

Comment:
```suggestion
  reserved 2; // tx_id was removed in v0.14
```

Reply:
I suggest just removing it and reserving the field number for backward compatibility.

Reply:
Should "tx_id" also be reserved?

Reply:
Have reserved `2` at this stage